src/main/java/org/sefglobal/scholarx/service/MailInstance.java

```diff
@@ -0,0 +1,29 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.mailer.MailerBuilder;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Why do we need to make this a service?

Reply:
Actually, I had no idea why this `Service` annotation was in the other classes that were located in the service folder. (I thought it was needed, so I added it. ) If it's not necessary will remove the annotation.
@jayasanka-sack

Reply:
Please do some research. Remove if not needed. 

(You should be responsible for what you write. Do not code blindly ;) )

Reply:
I think this should go to the utils directory. Also, we can combine this `MailInstance` class and the things in the `EmailUtil`.  WDYT? @jayasanka-sack

Reply:
Fixed with [b7fe478](https://github.com/sef-global/scholarx/pull/345/commits/b7fe4789eab6ff7c620f2bd33ef4168506e110ff)